app/config/question-bank.js

```diff
@@ -2324,6 +2324,93 @@ const questionBank = {
           },
           yarKey: "separatorType",
         },
+        {
+          key: 'gantry',
+          order: 142,
+          title: 'Do you want to add a gantry?',
+          hint: {
```

Comment:
hard coded text so cant approve. This needs to come from the database

Reply:
yup you're right